lib/quicly.c

```diff
@@ -2417,13 +2500,26 @@ static int handle_ack_frame(quicly_conn_t *conn, size_t epoch, quicly_ack_frame_
         packet_number += frame->gaps[gap_index];
     }
 
+    /* update bytes in pipe */
+    conn->egress.cc.bytes_in_flight -= conn->egress.cc.this_ack.nbytes;
+
+    /* loss-detection, as well as verifying  */
     quicly_loss_on_ack_received(&conn->egress.loss, frame->largest_acknowledged,
                                 last_packet_sent_at <= now && packet_number >= frame->largest_acknowledged
                                     ? (uint32_t)(now - last_packet_sent_at)
                                     : UINT32_MAX);
+    int rto_verified = quicly_loss_on_packet_acked(&conn->egress.loss, frame->largest_acknowledged);
```

Comment:
what is verified here?

Reply:
It is a flag telling if `OnRetransmissionTimeoutVerified` needs to be invoked. Based on https://tools.ietf.org/html/draft-ietf-quic-recovery-12#section-3.5.6.